### PR TITLE
Default text state for logged in user with a state 

### PR DESCRIPTION
**Pull request by cklosowski**
#3906

### CODE REVIEW

File: includes/checkout/template.php [hunk 1]
```diff
@@ -396,24 +396,25 @@ function edd_default_cc_address_fields() {
...
+			<select name="card_state" id="card_state" class="card_state edd-select<?php if( edd_field_is_required( 'card_state' ) ) { echo ' required'; } ?>">
+				<?php
+					foreach( $states as $state_code => $state ) {
+						echo '<option value="' . $state_code . '"' . selected( $state_code, $selected_state, false ) . '>' . $state . '</option>';
+					}
+				?>
+			</select>
+			<?php else : ?>
+			<?php $customer_state = ! empty( $customer['address']['state'] ) ? esc_attr( $customer['address']['state'] ) : ''; ?>
+			<input type="text" size="6" name="card_state" id="card_state" class="card_state edd-input" value="<?php echo $customer_state; ?>" placeholder="<?php _e( 'State / Province', 'easy-digital-downloads' ); ?>"/>
```
Comment (pippinsplugins) [inline includes/checkout/template.php]: Add esc_attr() here please

Comment (cklosowski) [inline includes/checkout/template.php]: @pippinsplugins it's done on the line above
`$customer_state = ! empty( $customer['address']['state'] ) ? esc_attr( $customer['address']['state'] ) : '';`

Comment (pippinsplugins) [inline includes/checkout/template.php]: Escaping should always be as late as possible so let's please move it to the HTML

**Comment by cklosowski** [comment]
@pippinsplugins ready for review again.

**Pull request merged**